app/handlers/createHighlightVideo.py

```diff
@@ -1,8 +1,41 @@
 #!/usr/bin/env python
 
+import os
 import tornado.web
+import tornado.escape
+from trafficcloud.video import create_highlight_video, get_framerate
```

Comment:
Maybe move video.py into traffic_cloud_utils so that we can get closer to removing trafficcloud folder